src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -10,17 +12,35 @@ import './SidebarNavSign.scss';
 export type Props = PlainButtonProps & {
     intl: IntlShape;
     status?: string;
+    targetingApi?: {
+        canShow: boolean;
+        onClose: () => void;
+        onComplete: () => void;
+        onShow: () => void;
+    };
 };
 
-export function SidebarNavSign({ intl, status, ...rest }: Props) {
+export const PlaceholderTooltip = ({ children }: { children: React.ReactNode }) => children;
+
+export function SidebarNavSign({ intl, status, targetingApi, ...rest }: Props) {
+    const isTargeted = targetingApi && targetingApi.canShow;
+    const FtuxTooltip = isTargeted ? TargetedClickThroughGuideTooltip : PlaceholderTooltip;
```

Comment:
Is the `PlaceholderTooltip` needed? Seems like if `targetingApi.canShow` is false, then the tooltip should not show already, do we need to not render it also?

Reply:
The `TargetedClickThroughGuideTooltip` doesn't check to see if `targetingApi` is defined before trying to use it. I figured that pre-empting its rendering was the safest way to handle it.